tests/meta-mender-ci/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -11,7 +11,7 @@ do_install_ptest_base() {
 # Produce custom level logs when MENDER_CI_LOGLEVEL is enabled
 do_install:append:class-target:mender-image:mender-systemd() {
     if ${@'true' if d.getVar('MENDER_CI_LOGLEVEL') else 'false'}; then
-        sed -i "s|ExecStart=.*|ExecStart=/usr/bin/mender --log-level ${MENDER_CI_LOGLEVEL} daemon|" \
+        sed -i "s|ExecStart=.*|ExecStart=/usr/bin/${MENDER_CLIENT} --log-level ${MENDER_CI_LOGLEVEL} daemon|" \
```

Comment:
Nitpick: Could use `test:` prefix for this commit.

Reply:
Great nit :tada: